src/api/route-services/__mocks__/projects.js

```diff
@@ -8,6 +8,24 @@ const mockGetProject = jest.fn((projectUuid) => new Promise((resolve) => {
   resolve(OK());
 }));
 
+const mockGetProjects = jest.fn(() => new Promise((resolve) => {
```

Comment:
Where is this being used? 
I am confused because the test `GetProjects gets all the projects` has the same list, but I understand that is calling the real function

Reply:
The functions under `route-services/__mocks__/` are mostly being used in tests for `routes`